packages/core/src/external/athenahealth/index.ts

```diff
@@ -181,6 +185,57 @@ class AthenaHealthApi {
     }
   }
 
+  async getPatientViaSearch({
+    cxId,
+    patientId,
+  }: {
+    cxId: string;
+    patientId: string;
+  }): Promise<PatientResource | undefined> {
+    const { log, debug } = out(`AthenaHealth search - AH patientId ${patientId}`);
+    try {
+      const data = {
+        _id: this.createPatientId(patientId),
+        "ah-practice": this.createPracticetId(this.practiceId),
+      };
+      const patienSearchtUrl = "/Patient/_search";
+      const response = await this.handleAxiosRequest(() =>
+        this.axiosInstanceFhirApi.post(patienSearchtUrl, this.createDataParams(data))
+      );
+      if (!response.data) throw new Error(`No body returned from ${patienSearchtUrl}`);
```

Comment:
hypernit: `nobody`

Reply:
It's referring to the req `body` not a person.

Reply:
ah, i see! i was confused there haha